src/menu/menu.service.ts

```diff
@@ -67,14 +67,7 @@ export class MenuService {
     } = dto;
 
     const menuItem = await this.prisma.$transaction(async () => {
-      await this.prisma.image.update({
-        where: {
-          id: imageId,
-        },
-        data: {
-          isAssigned: true,
-        },
-      });
+      await this.mediaService.updateImage(imageId, { isAssigned: true });
```

Comment:
you have specific method for this in media service. no need to expose method like that. anyone can change anything and hard to protect consistency